source/slang/lexer.h

```diff
@@ -83,15 +83,7 @@ namespace Slang
 
         TokenList lexAllTokens();
 
-        // Begin overriding the reported locations of tokens,
-        // based on a `#line` directives
-        void startOverridingSourceLocations(SourceLoc loc);
```

Comment:
It is nice to see these go away. It seems terrible for the lexer to even have to understand that `#line` exists.